test/test_jit.py

```diff
@@ -10154,7 +10154,7 @@ def foo(self):
                 m_orig = M(type)
                 m_import = self.getExportImportCopy(m_orig)
                 # check to make sure the storage wasn't resized
-                self.assertTrue(m_orig.param.storage().size() == 25)
+                self.assertTrue(m_orig.param.storage().size() == 25 * m_orig.param.element_size())
```

Comment:
Oof. This one will be difficult to warn against, without just removing the size() method entirely from storage(). Hopefully not too many people affected by it.

Reply:
Hmm, well we could just rename it to `nbytes`

Reply:
yeah let's rename it